index.html

```diff
@@ -3007,69 +3007,299 @@ <h3>Truncating or limiting the length of strings</h3>
 	<p class="advisement">Specifications SHOULD NOT limit the size of data fields unless there is a specific practical or technical limitation.</p>
```

Comment:
The previous paragraph talks about "data structure or text field".

What's a 'data field'?  Is it a string in a database or a string presented to the reader? The text that follows appears to indicate the latter, since it refers to CSS text-overflow.  My first impression was the former.